security/auto-host-endpoints.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Protect Kubernetes nodes
+description: Protect Kubernetes nodes with host endpoints managed by {{site.prodname}}
+---
+
+### Big picture
+
+Secure Kubernetes nodes with host endpoints managed by {{site.prodname}}.
+
+### Value
+
+{{site.prodname}} can automatically create host endpoints for your Kubernetes nodes. The lifecycle of these host endpoints are managed by {{site.prodname}} in response to an evolving cluster to ensure policy protecting your nodes is always enforced.
+
+### Features
+
+This how-to guide uses the following Calico features:
+- **HostEndpoint**
+- **Profile**
+- **KubeControllersConfiguration**
+- **GlobalNetworkPolicy**
+
+### Concepts
+
+### Host endpoints
+
+Each host has one or more network interfaces that it uses to communicate externally. You can use {{site.prodname}} network policy to secure these interfaces (called host endpoints).
+{{site.prodname}} host endpoints can have labels, and they work the same as labels on workload endpoints. The network policy rules can apply to both workload and host endpoints using label selectors.
+
+Host endpoints come in two types: *named* and *wildcard*. Named host endpoints secure a specific interface such as "eth0", and are created by setting `interfaceName: <name-of-that-interface>` -- for example, `interfaceName: eth0`.
+Wildcard host endpoints secure _all_ of the hosts interfaces non-workload interfaces. They are created by setting `interfaceName: "*"`.
+
+### Automatic host endpoints
+
+In order to protect a Kubernetes node, a host endpoint has to be created for it. This is fine for a few cluster nodes but for larger clusters of hundreds or thousands of nodes, some automation is required.
+{{site.prodname}} can automatically create host endpoints for your Kubernetes nodes when they are created, and similarly tear down the host endpoints when the Kubernetes nodes leave the cluster.
+
+{{site.prodname}} creates a wildcard host endpoint for each node, with the host endpoint containing the same labels and IP addresses as its corresponding node.
+{{site.prodname}} will ensure these managed host endpoints maintain the same labels and IP addresses as its node by periodic syncs.
+This means that policy targetting these automatic host endpoints will function correctly with the policy put in place to select those nodes, even if over time the node's IPs or labels change.
+
+Automatic host endpoints are differentiated from other host endpoints by the label `projectcalico.org/created-by: calico-kube-controllers`.
+Enable or disable automatic host endpoints by configuring the default KubeControllersConfiguration resource.
+
+### Profiles
```

Comment:
I suggest skipping this profiles sub-section.  This is more like implementation detail rather than concepts you need to understand to follow the how-to.  We don't for example explain in the k8s network policy how-to that we create workload endpoints for pods, and give them profiles for default allow and for the namespace they are in.

Reply:
Removed